### PR TITLE
fix: add_stable_liquidity error and edit build script

### PR DESCRIPTION
**Pull request by lucky7323**
- add `ft_transfer_call` before `add_stable_liquidity` to prevent E21 error (token register)
- edit build script to build multiple networks
- remove unused imports 
- Gas amount and attached deposit amount optimization

### CODE REVIEW

**Review by koe7 — APPROVED** [review]
Thxxxx!! :)

File: ft/src/ref_function.rs [hunk 1]
```diff
@@ -10,17 +11,32 @@ use std::convert::TryFrom;
...
+                near_sdk::AccountId::try_from(REF_CONFIG.ref_address.to_string()).unwrap(),
+                amount.into(),
+                None,
+                "".to_string(),
+            )
+            .then(
+                ext_ref_finance::ext(
+                    near_sdk::AccountId::try_from(REF_CONFIG.ref_address.to_string()).unwrap(),
+                )
+                .with_attached_deposit(ONE_NEAR)
```
[the remark this answers is not in the document]
Comment (lucky7323) [reply ft/src/ref_function.rs:27]: right. I'll fix thx

[the remark this answers is not in the document]
Comment (lucky7323) [reply ft/src/ref_function.rs]: This is fixed. And attached gas amount  is also optimized.

**Comment by lucky7323** [comment]
e2e test result: https://explorer.mainnet.near.org/transactions/6yA2SNNE7NAPSVos12oHhU8gZToUS2qoMYVtbGzSafxi

**Pull request merged**